fedmsg_meta_fedora_infrastructure/tests/fedimg.py

```diff
@@ -26,14 +26,13 @@
 from .common import add_doc
 
 
-class TestImageUpload(Base):
-    """ These messages are awarded when an image starts has started,
-    completes, or fails. """
+class TestImageUploadStart(Base):
+    """ These messages are awarded when an image upload has started. """
```

Comment:
While we're here, could you add some more information to these docstrings about what fedimg does?  That'll make the fedmsg.com docs more readable.